packages/api/src/webhooks/cannon.ts

```diff
@@ -184,7 +184,7 @@ export default class WebhookCannon {
 
       // sign payload if there is a webhook secret
       if (webhook.sharedSecret) {
-        let signature = sign(params.body, webhook.sharedSecret);
+        let signature = await sign(params.body, webhook.sharedSecret);
```

Comment:
I think the `sign` function can be made non-async instead, since it's code has no `async/await` logic

Reply:
need the await or the signature would be `[Promise Object]` , try it out 🤯

Reply:
I think that's because of the `async` in the `sign` function declaration [here](https://github.com/livepeer/livepeer-com/blob/18bdffc0d95d1aa3ecd779b7e8a817c834eff14b/packages/api/src/controllers/helpers.js#L18), then it forces the result to be a `Promise`. Since the implementation has no `await` though, I think that `async` can be removed and the typing would be just `string` here even without the `await`. That because `crypto.Hmac.digest` also returns a `string`, or at least thats what [the docs](https://nodejs.org/api/crypto.html#crypto_hmac_digest_encoding) say.